bigtable/client/idempotent_mutation_policy.h

```diff
@@ -54,6 +54,15 @@ class SafeIdempotentMutationPolicy : public IdempotentMutationPolicy {
   std::unique_ptr<IdempotentMutationPolicy> clone() const override;
   bool is_idempotent(google::bigtable::v2::Mutation const&) override;
 };
+
+/// Implements a policy that retries all mutations.
```

Comment:
It would be good to warn the user here about what this means. If a non-idempotent mutation gets retried due a transient error, they will end up with multiple copies of the data with different timestamps. If adding cells, that is.